notification-sdk/src/main/java/org/sunbird/notification/email/Email.java

```diff
@@ -295,6 +295,26 @@ private boolean sendEmail(Session session, MimeMessage message) {
     return response;
   }
 
+  public boolean sendMail(
+          List<String> emailList, String subject, String body, List<String> ccEmailList, List<String> bccList) {
+    boolean response = true;
+    Session session = getSession();
+    try {
+      logger.debug("Value of from Email Value : " + fromEmail);
+      MimeMessage message = new MimeMessage(session);
+      addRecipient(message, Message.RecipientType.TO, emailList);
+      addRecipient(message, Message.RecipientType.CC, ccEmailList);
+      addRecipient(message, Message.RecipientType.BCC, bccList);
+      setMessageAttribute(message, fromEmail, subject, body);
+      response = sendEmail(session, message);
+      logger.debug("Status of Email Sent is : " + response);
+    } catch (Exception e) {
+      response = false;
+      logger.error("Exception occured during email sending " + e, e);
```

Comment:
You just need to concatenate e.getMessage() with String -- not the entire exception object

Reply:
This is done @karthik-tarento, pls verify.